src/admin_websocket.rs

```diff
@@ -208,12 +216,52 @@ impl AdminWebsocket {
         }
     }
 
+    #[cfg(feature = "client_signing")]
+    pub async fn authorize_signing_credentials(
```

Comment:
Technically doesn't have to be under this feature, you could still use this... Undecided about even having a feature for this. I'm not sure why you'd disable it.